raw_socket_listener/listener.go

```diff
@@ -682,12 +682,6 @@ func (t *Listener) readRAWSocket() {
 }
 
 func (t *Listener) buildPacket(packetSrcIP []byte, packetData []byte, timestamp time.Time) *packet {
```

Comment:
I feel like this was removed by accident?

Reply:
No, I think it's useless